src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseAmendmentServiceTests.cs

```diff
@@ -298,7 +298,24 @@ public async Task CreateReleaseAmendment()
                     ReleaseVersion = originalReleaseVersion,
                     ReleaseVersionId = originalReleaseVersion.Id,
                     File = dataFile1,
-                    FileId = dataFile1.Id
+                    FileId = dataFile1.Id,
+                    FilterSequence = new List<FilterSequenceEntry>
```

Comment:
Assuming `FilterSequence` (et al.) are concrete type properties rather than abstractions, you can shorthand the instantiations to simply use `new()` as the type is always inferred.

Reply:
I've used the new collection syntax, so `FilterSequence = [ ... ]`